docs/notifications.md

```diff
@@ -21,18 +38,16 @@ Use the 'Test' button to send a test notification and ensure that your channel i
 
 ![A correctly configured Slack webhook will send a notification like this](img/test-notification.png "Correctly configured webhook will send a notification like this")
 
-### Email
+## Email
 
 To receive email notifications about Semgrep findings on pull requests and code pushes, visit [Dashboard > Integrations](https://semgrep.dev/manage/integrations) and select 'Add integration' or 'Setup First Integration,' and then choose 'Email'. Enter your email address, give the channel a name of your choosing, and then click 'Save'.
 
 On each scan that has at least one finding, you will receive one email from Semgrep with a summary of all of the findings from that scan.
 
-### Pull request comments
+## GitHub pull request comments
 
-<!-- prettier-ignore-start -->
 !!! info
```

Comment:
Do we still need this info box given that the subheading directly above already includes “GitHub?”

Reply:
good point!